release/pkg/test/testdata/main-bundle-release.yaml

```diff
@@ -463,6 +463,15 @@ spec:
         uri: https://release-bucket/artifacts/v0.0.0-dev-build.0/cluster-api-provider-nutanix/manifests/infrastructure-nutanix/v1.1.1/metadata.yaml
       version: v1.1.1+abcdef1
     packageController:
+      bundle:
+        arch:
+        - amd64
+        - arm64
+        description: Container image for eks-anywhere-packages-bundles image
+        imageDigest: sha256:0123456789abcdef0123456789abcdef0123456789abcdef0123456789abcdef
+        name: eks-anywhere-packages-bundles
+        os: linux
+        uri: public.ecr.aws/release-container-registry/eks-anywhere-packages-bundles:v1-25-latest-eks-a-v0.0.0-dev-build.1
```

Comment:
For some reason, the release channel isn't the vb kubeVersion and this -eks-a-v* just gets appended on the end

Reply:
Release channel is `1-21, 1-22, etc`. If you want the `major.minor` kubeversion, then you can do
```
kubeVersion[1:strings.LastIndex(kubeVersion, ".")]
```

Reply:
Release channel is what I want for this. I just don't want the `-eks-a-v0.0.0-dev-build.1` thing. I noticed the release channel wasn't incrementing is what I was seeing there.

Reply:
Well, I fixed the tag format, but still getting the same release channel and kubeVersion each call